src/components/Navbar/NavigationDrawer/ThemeSwitcher.module.scss

```diff
@@ -0,0 +1,99 @@
+@use 'src/styles/theme';
+
+.triggerContainer {
+  width: 100%;
+  justify-content: center;
+  border: 1px solid var(--color-borders-hairline);
+}
+
+.popoverContent {
+  padding: var(--spacing-small);
+  min-width: unset !important;
+  max-width: calc(var(--spacing-large) * 8);
+  width: 100%;
+  box-shadow: var(--shadow-flat);
+
+  @include theme.dark {
+    box-shadow: none;
+    border: 1px solid var(--color-borders-hairline);
+  }
+}
+
+.popoverItem {
+  padding: var(--spacing-xsmall) var(--spacing-medium);
```

Comment:
For all new css, we should use pixel values not rem as per agreenment with Talha to make sure what is on Figma is 100% the same pixel values in the code base. We already have pixel values suffixed with `-px` e.g. `--spacing-xxsmall-px` etc. Please apply to entire file.